app/util/analytics.py

```diff
@@ -22,9 +11,14 @@
 SUBJOB_EXECUTION_FINISH = 'SUBJOB_EXECUTION_FINISH'
 SUBJOB_EXECUTION_START = 'SUBJOB_EXECUTION_START'
 
-_analytics_logger = None
-_eventlog_file = None
-_event_id_generator = Counter()
+_event_log = None
+
+
+def write_to_log_cache(event):
+    """
+    :type event: dict
+    """
+    # _log_cache.append(event)
```

Comment:
It was an easter egg I left for reviewers. Congratulations, you win a prize :sparkles: :gift_heart: 

... actually it was me being sloppy. I left in code I abandoned during my change :blush: 
good catch :smile: